examples/nemo/eg4/dawn_script.py

```diff
@@ -36,58 +36,55 @@
 '''Example wrapper to run PSyclone generated SIR code in DAWN.'''
 
 import textwrap
-import sys
-import argparse
-import ctypes
 import os.path
 from optparse import OptionParser
-from ctypes import *
-from config import __dawn_install_module__, __dawn_install_dawnclib__
+from ctypes import c_char_p, CDLL
+from config import __dawn_install_dawnclib__
 from dawn import *
 from dawn import sir_printer
```

Comment:
Why do we need to import `sir_printer` given that we do `import *` just above?

Reply:
There is a wildcard import because the generated PSyclone code needs to be added to this script (it does not run by itself) so we do not know which dawn classes and methods are going to be used beforehand.

I don't know why we need to import sir_printer but the script fails if it is not included this way. I've added a comment.